controllers.py

```diff
@@ -112,6 +116,11 @@
         "disable-enable-tserverUIPort": Suite(
             workloads.workloads["yugabyte-operator"]["disable-enable-tserverUIPort"], "test-yugabyte-operator/test/time-travel-tserverUIPort.yaml", sieve_modes.TIME_TRAVEL),
     },
+    "nifikop-operator": {
+        "change-config": Suite(
+            workloads.workloads["nifikop-operator"]["change-config"], "test-nifikop-operator/test/atomic-1.yaml", sieve_modes.ATOM_VIO, oracle_config={"interest_objects": [{"rtype": "pod", "namespace": "default", "name": "simplenifi-1.*"}], "effect_to_check": ["Create"]}
```

Comment:
For this example, the default effect_to_check ("Delete" only) is enough to find the bug, so we do not need to add "Create" for this test case

Reply:
Since the bug is related to recreate (delete+create) the pod, so I think we can keep the create aspect to show out the full recreate inconsistency.